tests/utils/test_urlutils.py

```diff
@@ -227,3 +230,32 @@ def test_url_host(self):
         """Test with an URL with no path."""
         url = QUrl('http://qutebrowser.org/')
         assert urlutils.filename_from_url(url) == 'qutebrowser.org.html'
+
+
+class TestSchemeOrder:
```

Comment:
pytest's `pytest.mark.parametrize` makes it a lot easier to write tests like this, where the test cases are very similar to each other (code untested):

```python
@pytest.mark.parametrize('scheme_order, expected', [
    (['http'], 'http://example.com'),
    (['https'], 'https://example.com'),
    (['https', 'http'], 'https://example.com'),
])
def test_scheme_order(self, scheme_order, expected, monkeypatch, config_stub):
    init_config_stub(config_stub, 'naive', scheme_order=scheme_order)
    monkeypatch.setattr('qutebrowser.utils.urlutils.config', config_stub)
    assert urlutils.fuzzy_url('example.com').toDisplayString() == expected
```

However it doesn't matter much here - I'm working on many more tests for `urlutils` anyways, so I'll just merge those into the other `fuzzy_url` tests then probably.